jax/_src/scipy/stats/multinomial.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import scipy.stats as osp_stats
+from jax import lax
+from jax._src.numpy import lax_numpy as jnp
+from jax._src.numpy.util import _wraps, _promote_args_inexact
+from jax._src.scipy.special import gammaln, xlogy
+
+@_wraps(osp_stats.multinomial.logpmf, update_doc=False)
+def logpmf(x, n, p):
+  """JAX implementation of scipy.stats.multinomial.logpmf."""
+  x = jnp.array(x.astype(int))
```

Comment:
Rather than unconditionally converting `x` to integer type, we should error if the `x` input is not integer typed to begin with (cf. discussion at https://github.com/google/jax/pull/11879#discussion_r955320248)

Reply:
Ah, that is much better. What should the error message be like? Right now it seems a bit- unprofessional?